src/main/java/com/impactupgrade/nucleus/service/segment/XeroAccountingPlatformService.java

```diff
@@ -425,6 +429,46 @@ protected String getExceptionDetails(Exception e) {
     }
 
     // Mappings
+    protected Contact toOrgContact(CrmAccount crmAccount, Optional<CrmContact> crmContact) {
+        if (crmAccount == null) {
+            return null;
+        }
+        Contact contact = new Contact();
+        contact.setName(crmAccount.name);
```

Comment:
Aha, probably the root issue. For standard household accounts, shouldn't this be using the old way of doing things, primarily taking the Contact's data? We're only needing the Account name/info when it's an Organization type of account.

Let's verify that with DR first...